portfolio/src/main/webapp/about.css

```diff
@@ -0,0 +1,132 @@
+html, body {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+}
+
+#header {
+    display: flex;
+    flex-direction: row;
+    padding: 10px;
+    align-items: center;
+    background-color: rgb(192, 192, 192);
+    background-image: linear-gradient(rgb(192, 192, 192), rgb(129, 129, 129));
+}
+
+#header img {
+    height: 75px;
+    width: 100px;
+}
+
+#truth_and_lies_game {
+    display: flex;
+    flex-direction: column;
+}
+
+#questions {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.truth_and_lies_question {
+    display: flex;
+    flex-direction: column;
+    width: 23%;
+    margin-right: 5%;
+    margin-left: 5%;
+}
+
+.truth_and_lies_question img {
+   width: 100%; 
+}
+
+input[type=radio] {
+    opacity: 0;
+}
+
+input[type=radio]:checked + img {
+  border: 1px solid #fff;
+  box-shadow: 0 0 3px 3px #090;
+}
+
+#message, #footer {
+    flex-shrink: 0;
+}
+
+#message {
+    align-content: center;
+    background-color: yellow;
+    width: 100%;
+}
```

Comment:
Delete these references to "message" ID.

Reply:
Removed message